src/components/CurriculumComponents/CurriculumDownloads/CurriculumDownloads.test.tsx

```diff
@@ -1,13 +1,12 @@
 import { useRef } from "react";
 import { useRouter } from "next/router";
 import userEvent from "@testing-library/user-event";
-import { waitFor } from "@testing-library/dom";
+import { act } from "react-dom/test-utils";
```

Comment:
@BTitterington why are we using `import { act, waitFor } from "@testing-library/react";` elsewhere

Reply:
@orangemug - Do you mean in this PR or elsewhere in codebase? I changed `waitFor` to `act` as it seemed like the more appropriate as it is waiting on state changes (maybe that's not right?). The change was also triggered by this warning `Warning: An update to ForwardRef(LinkComponent) inside a test was not wrapped in act(...)`.